src/components/AdminDashboard/CompleteIncident.js

```diff
@@ -125,6 +130,33 @@ const CompleteIncident = props => {
             <br />
           </>
         )}
+
+        {/* Title */}
+        {!editing ? (
+          <div className="dropdown-text-wrap">
+            <p className="complete-incident-dropdown-titles-bold">
+              Title:
+            </p>
+            <p>{incident.title || '(none)'}</p>
+          </div>
+        ) : (
+          <>
+            <label htmlFor="description" className="label">
```

Comment:
Should this be htmlFor="title" ?

Reply:
Actually looking closer, a bunch of the htmlFors don't match their respective input fields, not just this one.

Reply:
That's because I'm sloppy. Fixing that now... actually I'm going to remove the htmlFors and put the inputs inside the label tags.

Reply:
After going through this component more thoroughly there's some weird things going on with prop-drilling, and the handleSubmit function is weird too...